contracts/zero-ex/contracts/src/features/native_orders/NativeOrdersInfo.sol

```diff
@@ -168,8 +168,9 @@ abstract contract NativeOrdersInfo is
                 orderInfo: orderInfo
             })
         );
-        isSignatureValid = order.maker ==
-            LibSignature.getSignerOfHash(orderInfo.orderHash, signature);
+        isSignatureValid =
+            (order.maker == LibSignature.getSignerOfHash(orderInfo.orderHash, signature)) ||
```

Comment:
Is it worth using a temporary variable for `LibSignature.getSignerOfHash(orderInfo.orderHash, signature)`?

Reply:
same below..

Reply:
yes this should definitely only be called once